blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/RollupRunnable.java

```diff
@@ -98,6 +98,11 @@ public void run() {
             Rollup rollup = null;
             RollupType rollupType = RollupType.fromString((String) rollupTypeCache.get(
                     singleRollupReadContext.getLocator(), MetricMetadata.ROLLUP_TYPE.name().toLowerCase()));
+
+            if (rollupType == null) {
```

Comment:
Will `rollupType` ever be `null` here? Because rollupTypeCache, will return `BF_BASIC` if its not `null` right?